src/libcrun/cgroup.c

```diff
@@ -2828,6 +2828,26 @@ update_cgroup_v2_resources (runtime_spec_schema_config_linux_resources *resource
         return ret;
     }
 
+  /* Write unified resources if any.  They have higher precedence and override any previous setting.  */
```

Comment:
>  They have higher precedence and override any previous setting. 

Should this be in the spec?

Reply:
it is just to get started implementing `unified` assuming the spec configuration doesn't have any incompatible setting.  On the long run, it would be nice to detect any incompatible configuration and report the error.